### PR TITLE
split midfoot contacts into front and rear, and move y positions inward

### PR DESCRIPTION
**Pull request by rdeits**
* split midfoot contacts into `midfoot_front` (at 2/3 of the way from heel to toe) and `midfoot_back` (at 1/3 of the way from heel to toe)
* move the y poses of the contact points IN by 1cm each. @kuindersma has already tested this on the robot. We're hoping it will prevent foot roll, because the existing contact points slightly overestimate the width of the foot. 
* remove some additional contact point logic from `planZMPTraj` which is no longer necessary and was flat-out wrong for the case of rear-foot support walking.

### CODE REVIEW

File: examples/Atlas/test/runAtlasWalkingNarrowStairs.m [hunk 1]
```diff
@@ -45,7 +45,7 @@ function runAtlasWalkingNarrowStairs()
 nq = r.getNumPositions();
 
 % Generate boxes and IRIS safe terrain regions
-l = 0.22;
+l = 0.26;
```
Comment (rdeits) [inline examples/Atlas/test/runAtlasWalkingNarrowStairs.m]: stairs were made slightly longer to accommodate 2/3 of the foot, which more accurately reflects what we can do in hardware

**Comment by rdeits** [comment]
@kuindersma and I decided to keep the y values unchanged, so this just splits the midfoot into midfoot_front and midfoot_rear

**Comment by rdeits** [comment]
good to go?

**Comment by RussTedrake** [comment]
yup.  looks good.

**Pull request merged**